source/index.haml

```diff
@@ -81,25 +85,50 @@
       %ul.small-block-grid-2.large-block-grid-3.guest-list
 
         %li.brandi-grayson
-          %a.hover
+          .hover
             .overlay
               .text
                 YWCA Director of Employment Services
           %h2 Brandi Grayson
 
+
         %li.rob-harr
-          %a.hover
+          .hover
             .overlay
               .text
                 Sparkbox
           %h2 Rob Harr
+          .twitter-handle
+            %a(href="http://www.twitter.com/robertharr/")
```

Comment:
This could be on one line
`%a(href="http://www.twitter.com/robertharr/") @robertharr`